tests/baselines/reference/optionalChainWithInstantiationExpression1(target=es2020).js

```diff
@@ -16,5 +16,5 @@ a?.b.d
 
 
 //// [optionalChainWithInstantiationExpression1.js]
-a?.b.d;
+(a?.b).d;

```

Comment:
FWIW this is still illegal in the grammar (even though we parse it) as the type checker and node helpers aren't set up correctly to handle it. I guess it's fine to have this fixed, though, even if it's an error.